static/style/screen.css

```diff
@@ -8,16 +8,16 @@
 /******************************************************************************\
     General Styles
 \******************************************************************************/
-body { font: 80% "Trebuchet MS", sans-serif; text-align:left; }
+body { font: 90% "Gotham SSm A","Gotham SSm B",sans-serif; text-align:left; }
```

Comment:
Gotham font usage on this project won't work without a Cloud.typography CSS key.  You'll need to talk to Roger about incorporating one of our CSS keys to make this work, or consider just using Helvetica + Georgia as fallbacks.

Reply:
Jim added UCF Sans Serif Alt